html/search .html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Search Page</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Rowdies:wght@300;400;700&display=swap">
+    <link rel="stylesheet" href="../css/style-search.css">
+</head>
+<body>
+    <!-- start header -->
+    <header>
+        <img src="../images/Logo.png" alt="logo">
+        <a href="../index.html"><img src="../images/homeicon.png" alt="home icon"></a>
+    </header>
+    <!-- End header -->
+
+    <!-- start section search -->
+    <div class="search-bar">
+        <input type="text" placeholder="Type your word here">
+        <div class="search-icon">
+            <img src="../images/searchicon.png" alt="Search icon">
+        </div>
+    </div>
+    <!-- End section search -->
+
+    <!-- Start section image -->
+    <div class="sample-image">
+        <img src="../images/sample.png" alt="Sample image">
+    </div>
+    <!-- End section image -->
```

Comment:
**use the semantic element `section` instead of `div`**

**use the semantic element `figure` with an image here and do the same with the rest**